packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -99,6 +99,14 @@ export async function execCommand(command: string, options: SpawnOptions = {}) {
   return true;
 }
 
+export async function getNodeBinPath(entrypointDir: string) {
+  const { stdout } = await execAsync('yarn', ['bin'], {
```

Comment:
What about when the user doesn't have `yarn` installed?

 ```suggestion
  const { stdout } = await execAsync('npm', ['bin'], {
```

I think we should use `npm bin` instead since its more widely available.